pkg/controllers/management/clusterprovisioner/provisioner.go

```diff
@@ -725,8 +742,12 @@ func GetDriver(cluster *v3.Cluster, driverLister v3.KontainerDriverLister) (stri
 		}
 	}
 
+	if cluster.Spec.EKSConfig != nil {
+		return "EKS", nil
```

Comment:
This should not be hardcoded and should live along with the other driver types in apimgmtv3